packages/web/docs/src/app/layout.tsx

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { ReactNode } from 'react';
+import localFont from 'next/font/local';
+import { Layout, useConfig } from 'nextra-theme-docs';
+import { Head } from 'nextra/components';
+import { getPageMap } from 'nextra/page-map';
+import '@theguild/components/style.css';
+import '../components/navigation-menu/navbar-global-styles.css';
+import '../selection-styles.css';
+import { PRODUCTS } from '@theguild/components';
+import { getDefaultMetadata } from '@theguild/components/server';
+import { Footer } from '../components/footer';
+import { NavigationMenu } from '../components/navigation-menu';
+
+export default function RootLayout({ children }: { children: ReactNode }) {
+  return <HiveLayout>{children}</HiveLayout>;
+}
+
+export const metadata = getDefaultMetadata({
+  productName: PRODUCTS.HIVE.name,
+  websiteName: 'Hive',
+  description:
+    'Fully Open-source schema registry, analytics and gateway for GraphQL federation and other GraphQL APIs',
+});
+
+const neueMontreal = localFont({
+  src: [
+    { path: '../fonts/PPNeueMontreal-Regular.woff2', weight: '400' },
+    { path: '../fonts/PPNeueMontreal-Medium.woff2', weight: '500' },
+    { path: '../fonts/PPNeueMontreal-Medium.woff2', weight: '600' },
+  ],
+});
+
+/**
+ * Alternative to `GuildLayout` for Hive and Hive Gateway websites.
+ */
+const HiveLayout = async ({ children }: { children: ReactNode }) => {
+  const [meta, ...pageMap] = await getPageMap();
+
+  const productsPage = pageMap.find(p => 'name' in p && p.name === 'products')!;
+  // @ts-expect-error -- this should be fixed in Nextra, without route, the collapsible doesn't work
+  productsPage.route = '#';
+
+  return (
+    <html
+      lang="en"
+      // Required to be set for `nextra-theme-docs` styles
+      dir="ltr"
+      // Suggested by `next-themes` package https://github.com/pacocoursey/next-themes#with-app
+      suppressHydrationWarning
+      className="font-sans"
+    >
+      <Head>
+        <style>{
+          /* css */ `
+          :root {
+            --font-sans: ${neueMontreal.style.fontFamily};
+          }
+          body {
+            --nextra-primary-hue: 67.1deg;
+            --nextra-primary-saturation: 100%;
+            --nextra-primary-lightness: 55%;
+            --nextra-bg: 17, 17, 17;
+          }
+          body.light, .light body {
+            --nextra-primary-hue: 191deg;
+            --nextra-primary-saturation: 40%;
+            --nextra-bg: 255, 255, 255;
+          }
+          ._tracking-tight,
```

Comment:
```suggestion
          .x\\:tracking-tight,
```
maybe? there is new prefix with migration to Tailwindcss v4

Reply:
I'm still getting underscores. Wanted to update but noticed that issue with nextra Prettier config blocking release.
I'll try bump it in a sec and recheck.

<img width="432" alt="image" src="https://github.com/user-attachments/assets/2d5548db-4ca8-469d-9172-da99d293d509">

Reply:
Ah it was not released, I will do a release